controllers/argocd/deployment_test.go

```diff
@@ -293,6 +293,18 @@ func TestReconcileArgoCD_reconcileDexDeployment_removes_dex_when_disabled(t *tes
 		deployment))
 }
 
+// Test when both Dex and keycloak are configured
+func TestReconcileArgoCD_reconcileDexDeployment_with_keycloak_configured(t *testing.T) {
+	restoreEnv(t)
+	logf.SetLogger(ZapLogger(true))
+	a := makeTestArgoCDForKeycloakWithDex()
+
+	templateAPIFound = true
```

Comment:
Do we need this?

Reply:
Thanks for the comment, this unit test got removed in the latest push as changes in `deployment.go` got removed.